### ISSUE TITLE
Header. Abbreviated language toggle should be aria-labelled

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#101]
### Description of issue

With screen readers, the EN language toggle is read as the word "en".

### Potential fix
- We can label mobile language toggle links with the full name of the language.

**Comment by shawnthompson** [comment]
Instead of using `aria-label` you should be using `class="sr-only"` to hide the full text of the language.

Reference: https://tailwindcss.com/docs/screen-readers

```
<a hreflang="fr" lang="fr" href="#"><span aria-hidden="true">FR</span><span class="sr-only">français</span></a>
```

This tell screen readers not to read the _FR_ text but it would read the _français_ text, which will not be visible on the page. 

> `hreflang`
Hints at the human language of the linked URL. No built-in functionality. Allowed values are the same as the global lang attribute.

[`<a>`: The Anchor element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a)
